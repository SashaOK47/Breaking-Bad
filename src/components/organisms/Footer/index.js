import style from "./styles.module.scss";

const Footer = () => {
  return (
    <footer className={style.footer}>
      <div className={style.container}>
        <time>{new Date().getFullYear()}</time>
        <p>@Aleksandr Okoronko</p>
      </div>
    </footer>
  );
};
export default Footer;
