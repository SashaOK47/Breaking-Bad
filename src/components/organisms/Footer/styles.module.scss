@import "../../../assets/styles/main.scss";
.footer {
  padding: 1.75rem 0;
  background-color: $white;
  .container {
    display: flex;
    justify-content: space-between;
    align-items: center;
  }
}
