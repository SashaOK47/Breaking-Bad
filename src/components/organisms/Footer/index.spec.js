import React from "react";
import Footer from "./index";

describe("Footer Component", () => {
  it("should render Footer component", () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
