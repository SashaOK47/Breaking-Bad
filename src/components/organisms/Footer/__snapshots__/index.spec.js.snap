// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render Footer component 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <time>
      2022
    </time>
    <p>
      @Aleksandr Okoronko
    </p>
  </div>
</footer>
`;
